### DISCUSSION TITLE
MMOT Sinkhorn

### DISCUSSION THREAD

**Discussion by harveymannering** [PythonOT/POT#652] [Q&A] [answered]
Is anybody able to describe how you might use the Sinkhorn functions in the library for the multiple marginal variety of the the Sinkhorn algorithm?  I am refering specifically to the algorithm described in [Computational Optimal Transport, Section 10.1](https://arxiv.org/pdf/1803.00567).  If this is not possible within this library could anyone point me to an implementation of MMOT Sinkhorn somewhere else? Thank you!

**Comment by rflamary** [comment, marked as answer]
Hello, Entropic MMOT has not been implemented yet in POT. 

I am not aware of other Python implementation. Still the algorithm is rather easy to implement and we encourage you to contribute it to POT if you do it.
